### PR TITLE
[Feature] added script to update version

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
This PR introduces a script to update release versions automatically across different files for main repo. For more details, please refer to ```README.md``` under ```scripts``` folder

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
N/A

## What to Check
N/A

## Other Information
N/A

### CODE REVIEW

**Review by JoshuaLai — DISMISSED** [review]

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+This folder contains a series of scripts that facilitates the release process for UI iOS team. In particular, these scripts are meant to update release version across different files such as README, info plist, pbxproj, etc.
+
+
+| Name                 | Area        | Link |
+|----------------------|-------------|------|
+| main-repo-updater.py | Main Repo   | [Go](https://github.com/Azure/communication-ui-library-ios/blob/feature/script-version-update/scripts/main-repo-updater.py)     |
+| TBA                  | CocoaPods   | TBA  |
+| TBA                  | QuickStarts | TBA  |
+| TBA                  | MS Doc      | TBA  |
+| TBA                  | Hero Sample | TBA  |
```
Comment (JoshuaLai) [inline scripts/README.md:14]: Not a big deal, but I am not sure if it will be a good idea to have the script for hero sample hero or in its own repo

Comment (jpeng-ms) [reply scripts/README.md]: thanks updated

**Review by JoshuaLai — DISMISSED** [review]

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The script would attempt to find current version from Info.plist by looking for key ```UILibrarySemVersion```. If current versions is identical to the new version, the version update would be skipped.
+
+## Usage:
+
+#### Main Repo
+
+1. make sure this script is located under ```communication-ui-library-ios/scripts``` folder
+2. make sure you have ```python3``` installed
+3. run ```python3 main-repo-updater.py -v NEW_VERSION``` on your terminal where ```NEW_VERSION``` should be the upcoming release version or ```python3 main-repo-updater.py --version NEW_VERSION```
```
Comment (jimchou-dev) [inline scripts/README.md:32]: Will there be separate script file for chat and callingWithChat in the future? Or will the same script be refactored to be used for all the packages?

Comment (vhuseinova-msft) [inline scripts/README.md:32]: `on your terminal where ```NEW_VERSION``` should be the upcoming release version` should be moved to the end of the sentence to show that both commands should have changed `NEW_VERSION` value.
Something like:
run ```python3 main-repo-updater.py -v NEW_VERSION``` or ```python3 main-repo-updater.py --version NEW_VERSION``` on your terminal where ```NEW_VERSION``` should be the upcoming release version

Comment (jpeng-ms) [reply scripts/README.md:32]: thanks!

Comment (jpeng-ms) [reply scripts/README.md:32]: I was thinking to expand this script to have a new param that refers to our SDKs:

```
python3 main-repo-updater.py --sdk chat --version 1.1.0
python3 main-repo-updater.py --sdk call --version 1.1.0
python3 main-repo-updater.py --sdk callwithcall --version 1.1.0

or 

python3 main-repo-updater.py -s chat -v 1.1.0
python3 main-repo-updater.py -s call -v 1.1.0
python3 main-repo-updater.py -s callwithcall -v 1.1.0
```

Comment (jimchou-dev) [reply scripts/README.md]: sounds good, so refactoring will be done in the future on the same script

**Review by jimchou-dev — APPROVED** [review]

File: scripts/README.md (hunk 2 above)
Comment (jpeng-ms) [reply scripts/README.md]: yes it will

**Review by JoshuaLai — APPROVED** [review]

**Review by vhuseinova-msft — APPROVED** [review]

**Pull request merged**